tests/integration/components/ember-popper-targeting-parent-test.js

```diff
@@ -25,6 +25,36 @@ test('it targets the parent', function(assert) {
   });
 });
 
+test('it passes ariaRole as role', async function(assert) {
```

Comment:
I wrote these tests with `async/await`, which is common in Ember, but not in this repo. I'm happy to (a) leave them as-is, (b) change them to `wait()` like the other tests, or (c) change all the other tests to `async/await` in a separate commit. Just let me know.